.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
I didn't see the package.json config. I prefer having babel config on their own file. This is actually from my work of adding ES2015 support that forgot to delete it. 

I just realized that we might not need both of them since you already set 

```js
query: {
    presets: ['es2015'],
},
```

on the babel-loader config.